app/controllers/points_controller.rb

```diff
@@ -10,15 +10,37 @@ def create
     end
   end
 
+  def update
+    @point = Point.find(params[:id])
+    @point.attributes = point_params.except(:image)
```

Comment:
I don't think you want to wholesale replace the `@point.attributes`. That would clear out any other properties on the `@point`.

I think you want `@point.assign_attributes(...)` here.